pkg/test-infra/cdc/recommend.go

```diff
@@ -91,7 +91,9 @@ func newCDC(ns, name string) *CDC {
 									"server",
 									fmt.Sprintf("--pd=%s", fmt.Sprintf("http://%s:2379", upstreamPDAddr)),
 									"--status-addr=0.0.0.0:8300",
+									"--log-level", "debug",
```

Comment:
What about making the log level as a configurable flag? You can add flags here https://github.com/pingcap/tipocket/blob/4f56639edf2b27712f87bb41a3d225236bef38e2/pkg/test-infra/fixture/fixture.go#L246